app/chain/index.ts

```diff
@@ -156,9 +194,11 @@ class Chain {
     async eventListenerStop() {
         logger.debug('[EventListenerStop] Stopping event listener...');
         if (this.unsubscribeEventListener) {
-            this.unsubscribeEventListener();
+            // TODO_CHECK I suspect some of the issues with the file are to do with this
```

Comment:
I agree with you.  We also need to change the `eventListenerStart` function.
```
this.unsubscribeEventListener = await xxxxx
```
Add `await` in the source code.
https://polkadot.js.org/docs/api/start/api.query.subs